docs/configuration.rst

```diff
@@ -0,0 +1,129 @@
+.. _configuration file:
+
+*************
+Configuration
+*************
+
+Robocop provides a wide set of rules with good defaults and is designed to work well out-of-the-box.
+Nevertheless, it is natural that some projects may have different needs and requirements, which influence the way the code is written.
+Thus, Robocop allows to configure many rules to fit these needs.
+It's done using CLI options and such configuration can be stored and loaded from file, as described below.
+
+Configuring rules
+=================
+
+Rules are configurable. Severity of every rule message can be changed (see more :ref:`severity-threshold`) and also some of the rules have
```

Comment:
Severity threshold is something different than configuring severity. You can configure severity as const, so it will always use configures value. Additionaly, some selected rules allow to configure 'dynamic' severity and thats severity threshold

Reply:
You're right. I mixed these two.